README.md

```diff
@@ -116,17 +124,38 @@ A state has the following fields:
 | criterion_scores  | [Score]               | The most recent score for each criterion. |
 | links             | [Link]                |             |
 
+#### GET /states
+
+This endpoint returns information about all 50 states and DC.
+
 #### GET /states/{code}
 
 This endpoint returns the state corresponding to the state code provided in the request. If no state with that code exists, it will return a 404 response code.
 
+#### PUT /states/{code} (UPCOMING)
+
+This endpoint updates the state corresponding to the state code provided in the request. It requires [authentication](#Authentication)
+and accepts a JSON body with the following format:
+
+|       Name        |          Type         |    Notes    |
+|-------------------|-----------------------|-------------|
+| innovative_idea   | String                | *Optional*. |
+| honorable_mention | String                | *Optional*. |
+| grade             | Integer               | *Optional*. One of (-1, 0, 1, 2, 3) |
+| category_grades   | Array<Object>         | *Optional*. An array whose elements are objects in the format `{category_id: ..., grade:...}`. Every `category_id` must be the id of an active category. Every `grade` must be an integer between -1 and 3.|
+| links             | Array<Object>         | *Optional*. An array whose elements are objects in the format `{category_id: ..., text: ..., url: ..., active: ...}`. See the [Links](#Links) documentation for information on each of these fields.|
+| scores             | Array<Object>         | *Optional*. An array whose elements are objects in the format `{category_id: ..., meets_criterion: ...}`. See the [Scores](#Scores) documentation for information on each of these fields.|
```

Comment:
@brittanylau This is related to what we were talking about on Slack -- this endpoint would create and update state grades, category grades, and links behind the scenes, but to the frontend it would all be one PUT command.

Reply:
Love it!!